theano/gof/opt.py

```diff
@@ -2053,12 +2014,39 @@ def print_profile(stream, prof, level=0):
         print(blanc, "  loop time", loop_t, file=stream)
         print(blanc, "  callback_time", callback_time, file=stream)
         if isinstance(lopt, LocalOptGroup):
-            lopt.print_profile(stream, (lopt.time_opts,
```

Comment:
From my analysis of removing commits, I suspect this call to print_profile could be the reason for slow down. Because, it all boils down to the commit where I did this. But, I am not able to figure out why that would happen. @lamblin could you please explain ?